enterprise/app/code/code_v2.tsx

```diff
@@ -389,7 +390,6 @@ export default class CodeComponentV2 extends React.Component<Props, State> {
 
     return this.getModel(path).then((model) => {
       this.setModel(path, model);
-      this.focusLineNumberAndHighlightQuery();
```

Comment:
did you mean to remove this / am i missing some other way that this winds up highlighting? or did it not even work